lib/bundles/inspec-compliance/api/login.rb

```diff
@@ -0,0 +1,150 @@
+# encoding: utf-8
+# author: Christoph Hartmann
+# author: Dominik Ricter
+# author: Jerry Aldrich
+
+module Compliance
+  class API
+    module Login
+      class CannotDetermineServerType < StandardError; end
+
+      def login(options)
+        raise ArgumentError, 'Please specify a server using `inspec compliance login https://SERVER`' unless options['server']
+
+        options['server_type'] = Compliance::API.determine_server_type(options['server'], options['insecure'])
+
+        case options['server_type']
+        when :automate
+          config = Login::AutomateServer.login(options)
```

Comment:
Nice separation!

Reply:
Thanks! Took some work that's for sure :sweat_smile: